Source/ASDisplayNode.mm

```diff
@@ -80,6 +80,11 @@ @implementation ASDisplayNode
 
 static std::atomic_bool storesUnflattenedLayouts = ATOMIC_VAR_INIT(NO);
 
+- (ASDisplayNodeMethodOverrides)methodOverrides
```

Comment:
I think we should put this method near other setters/getters (maybe [here](https://github.com/TextureGroup/Texture/blob/master/Source/ASDisplayNode.mm#L410)?). What do you think?

Reply:
Sounds good. It has been moved